rsconnect/main.py

```diff
@@ -244,6 +244,58 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+# inverts bool args if they are provided, otherwise returns None
+def env_management_callback(ctx, param, value):
```

Comment:
nit; type hinting is always appreciated.

```suggestion
def env_management_callback(ctx, param, value) -> Optional[bool]:
```

Reply:
I need to make this a habit, thanks for the reminder!